sat/cli/bootprep/input/session_template.py

```diff
@@ -186,6 +186,12 @@ def get_create_item_data(self):
             boot_set_api_data.update(boot_set_data)
             api_data['boot_sets'][boot_set_name] = boot_set_api_data
 
+            # Fetch full boot sets
+            boot_sets = self.boot_set
+            # Render the rootfs_provider_passthrough using Jinja2
+            rendered_rootfs = self.jinja_env.from_string(
+                        boot_sets[boot_set_name]['rootfs_provider_passthrough']).render(self.data)
```

Comment:
Exceptions also aren't caught here which will lead to tracebacks (e.g. if a variable used in a Jinja expression is not set).